### ISSUE TITLE
Add a section to load a pre-deployed program in solpg like what remix has done.

### ISSUE THREAD

**Issue by pooriagg** [solana-playground/solana-playground#159]
This a very important feature i think.

![Screenshot 2023-05-02 143135](https://github.com/solana-playground/solana-playground/assets/104061307/e26643fb-6a3a-441c-b6c8-e8c34067111f)

**Comment by acheroncrypto** [comment]
Yeah I agree, this would be super useful. One way to achieve this would be to fetch the Anchor IDL on-chain(https://github.com/solana-playground/solana-playground/issues/93) if it exists.

**Comment by pooriagg** [comment]
Yes ,also if we could't fetch the IDL or if that program is a native solana program we can ask user to copy their code to the ide.
